components/logins/src/error.rs

```diff
@@ -65,8 +65,8 @@ error_support::define_error! {
 
 #[derive(Debug, Fail)]
 pub enum InvalidLogin {
-    #[fail(display = "Hostname is empty")]
-    EmptyHostname,
+    #[fail(display = "Origin is empty")]
```

Comment:
Nit: A comment that this refers to the hostname field, which we'd like to rename to origin, could be helpful until we get around to renaming hostname to origin.